### ISSUE TITLE
How can I inject scalars in root graphql-module to be available to all children?

### ISSUE THREAD

**Issue by pixelmeister** [Urigo/graphql-modules#237]
I would like to use these scalars [https://github.com/okgrow/graphql-scalars](https://github.com/okgrow/graphql-scalars) in my modules. How can I inject them in the root graphql module so that all children do not have to import them?  I tried building a schema of them and then using the extraSchemas field (guessing that is what that field is for?) in the top most module, but that didn't seem to work.

**Comment by ardatan** [comment]
You can create a commonModule that has scalars with their resolvers, and import this commonModule in other modules.

**Comment by pixelmeister** [comment]
I was looking for a way of importing them for all modules (without each module importing), but perhaps the violates the spirit of modules.  I went ahead and made a common one to import and it did not work. I then downloaded the latest repo cde, built and used that version and it all worked. I'm guessing this commit:

[https://github.com/Urigo/graphql-modules/commit/76f0fc42a06a03f6fd0bdf4f1a7292e695000635](https://github.com/Urigo/graphql-modules/commit/76f0fc42a06a03f6fd0bdf4f1a7292e695000635)

fixed the issue I was having.  Any chance to get a new release published?  Thanks and thanks for the great work!

**Comment by ardatan** [comment]
We are going to release it in a few days :)

**Comment by ardatan** [comment]
You can use these canary versions in the meantime;
```
Successfully published:
 - @graphql-modules/core@0.3.0-alpha.9ce6ba88
 - @graphql-modules/di@0.3.0-alpha.9ce6ba88
 - @graphql-modules/epoxy@0.3.0-alpha.9ce6ba88
 - @graphql-modules/sonar@0.3.0-alpha.9ce6ba88
 - @graphql-modules/utils@0.3.0-alpha.9ce6ba88
```

**Comment by orefalo** [comment]
FYI, same issue. Glad is was discussed/resolved a few hours ago!

**Comment by pixelmeister** [comment]
The canary versions work great. Thanks!!

**Comment by ardatan** [comment]
Fixed in 0.2.18
